### ISSUE TITLE
[Bug]: theme has no effect on editor

### ISSUE THREAD

**Issue by tomaswarynyca** [mallowigi/material-theme-issues#377]
### Describe the bug

The editor looks like InteliJ's Dark theme! (intellij idea 2023.2)
Activated theme: Darker Material
![image](https://github.com/mallowigi/material-theme-issues/assets/41587659/daea7c86-f19c-4953-9564-fb82f177d651)

### To Reproduce

When I start the IDE it appears like this!
Note: activating or deactivating a style sheets plugin fixes it for as long as the IDE is open.

### Expected behavior

![image](https://github.com/mallowigi/material-theme-issues/assets/41587659/8ecb9297-3698-4031-9cfa-1b30e13a16a9)

### Material Theme Version

v8.9.1

### IDE

IntelliJ IDEA

**Comment by mallowigi** [comment]
Sorry, I don't understand the issue or the steps here... can you provide a video showing what happens ?

**Comment by tomaswarynyca** [comment]
The editor part has a colour at startup :/ 
[Screencast from 2023-07-29 09-58-37.webm](https://github.com/mallowigi/material-theme-issues/assets/41587659/f01929b8-7a17-4247-b629-4d716e9e41fc)

**Comment by tomaswarynyca** [comment]
Hi @mallowigi, were you able to understand the problem? 
It seems very strange to me :grimacing:

**Comment by mallowigi** [comment]
Might be related to the Settings Sync. Do you have it enabled?

**Comment by tomaswarynyca** [comment]
You mean the plugin? No
![image](https://github.com/mallowigi/material-theme-issues/assets/41587659/e6408f88-2acb-45f8-b914-d50e2f028a25)

**Comment by mallowigi** [comment]
Can you send me more information about your setup? OS, IDE, plugins...

**Comment by ddubrava** [comment]
Hi! I'm experiencing the same problem on Windows, while everything works fine on MacOS. The project explorer changes, but the color scheme remains the default one.

![image](https://github.com/mallowigi/material-theme-issues/assets/22116465/d173436c-3713-4d28-aed0-be8c02b09232)
![image](https://github.com/mallowigi/material-theme-issues/assets/22116465/f48de538-77c2-421f-b08a-118436bc156e)
![image](https://github.com/mallowigi/material-theme-issues/assets/22116465/d59bb1b6-7b4e-4aa4-8852-78c6344414c7)

It seems that the problem is specific to Material Darker, as Material Oceanic looks okay.

![image](https://github.com/mallowigi/material-theme-issues/assets/22116465/234fd639-149f-48d0-9e3f-d6127a271c70)

By the way, on my Mac, I have version 8.9.1, while on Windows, it's version 8.6.4, which is the latest available version

```
WebStorm 2023.1.4
Build #WS-231.9225.18, built on July 13, 2023
Licensed to Daniil Dubrava
You have a perpetual fallback license for this version.
Subscription is active until May 4, 2024.
Runtime version: 17.0.7+10-b829.16 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 2048M
Cores: 12
Registry:
    editor.focus.mode.color.light=474747
    ide.editor.tab.selection.animation=true
    ide.animate.toolwindows=true
    ide.new.editor.tabs.vertical.borders=true
    ide.experimental.ui=true
    ide.balloon.shadow.size=0
    editor.focus.mode.color.dark=474747

Non-Bundled Plugins:
    String Manipulation (9.10.0)
    ru.adelf.idea.dotenv (2023.1)
    com.mallowigi (90.0.0)
    mobi.hsz.idea.gitignore (4.5.1)
    com.chrisrm.idea.MaterialThemeUI (8.6.4)
```

**Comment by ddubrava** [comment]
ok, I fixed it by importing the settings (File - Manage IDE Settings - Import Settings). I believe that resetting everything fixes the issue

**Comment by ddubrava** [comment]
you are right; it happens due to settings sync. If I import settings (File - Manage IDE Settings - Import Settings), everything works fine (sync is disabled by default when you import settings). However, once I enable the sync, everything breaks again

**Comment by mallowigi** [comment]
I started using the sync settings and it happens to me as well. I'll investigate this.

**Comment by mallowigi** [comment]
It's actually very annoying. JetBrains started adding a combination of <theme,colorScheme> with sometimes weird values. I've started a settings sync from scratch and I can see some combinations like "Deep Ocean,Darcula"...

**Comment by mallowigi** [comment]
In that case, here's how to fix it:

When you have a misconfiguration between the Theme and the Color Scheme:
- Open Settings
- Go to Editor > Color Scheme
- Check that you are on the right color scheme (ex: if Moonlight, check that you are on Moonlight color scheme)
- If not, select Moonlight and click OK

But sometimes, there are also cases when a color scheme is duplicated on the user machine (it usually happens if you select a scheme bundled from the plugin, for example Night Owl, but don't have the plugin installed on the other machine. The settings sync will then download Night Owl but use Darcula colors). In that case, since it has the same name as the bundled color scheme, the IDE will pick that color scheme instead. 

In that case, in the color schemes screen, check for superfluous color schemes, then press on the settings menu and select "Delete" for all that are involved.

**Comment by mallowigi** [comment]
https://share.cleanshot.com/Hx0fHvCT

**Comment by mallowigi** [comment]
[IDEA-330252](https://youtrack.jetbrains.com/issue/IDEA-330252) Settings Sync is messing up the Material Theme plugin follow up
